frontend/src/lib/components/DefinitionPopover/DefinitionPopoverContents.tsx

```diff
@@ -69,6 +75,82 @@ export function VerifiedDefinitionCheckbox({
     )
 }
 
+export function PropertyStatusControl({
+    verified,
+    hidden,
+    showHiddenOption,
+    allowVerification,
+    onChange,
+    compact = false,
+}: {
+    verified: boolean
+    hidden: boolean
+    showHiddenOption: boolean
+    allowVerification: boolean
+    onChange: (status: { verified: boolean; hidden: boolean }) => void
+    compact?: boolean
+}): JSX.Element {
+    const copy = {
+        verified:
+            'Prioritize this property in filters and other selection components to signal to collaborators that this property should be used in favor of similar properties.',
+        visible: 'Property is available for use but has not been verified by the team.',
+        hidden: 'Hide this property from filters and other selection components by default. Use this for deprecated or irrelevant properties.',
+    }
+
+    const verifiedDisabledCorePropCopy = 'Core PostHog properties cannot be verified, but they can be hidden.'
```

Comment:
Oh, actually I think the UX so far has been that core PostHog properties are effectively always verified. Not sure who's the authority on this, I think @corywatilo has been involved with verification?

Reply:
Updated language

Reply:
Updated language